Makefile

```diff
@@ -129,8 +134,8 @@ endif
 .PHONY: bundle
 bundle: manifests kustomize
 	operator-sdk generate kustomize manifests -q
-	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
-	$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
+	cd deploy/base && $(KUSTOMIZE) edit set image authorino=$(AUTHORINO_IMAGE)
+	$(KUSTOMIZE) build install/crd | operator-sdk generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
```

Comment:
I understand your point about separating the "install" from the "deploy".
The `config` directory is part of the standard layout provided by the `operator-sdk` tool.
Potential issues I can see with separating the directories is that operator-sdk tool might have some assumptions on the directory and/or all configs being in the same parent directory. For example, this same `bundle` Make target is not properly generating the bundle now: you will see for example that the role permissions are not included in the generated bundle CSV, nor the CRDs. To fix this a workaround would need to be figured out.
There might be also another issues as a consequence of this that we are unaware of at the moment. For example quickly thinking I know there are CSV annotations specific to operator-sdk and I wonder if it might make assumptions on directory locations from that side.

In general in an operator-sdk based project moving away from the standard layout seems problematic.

Reply:
I think we need a concrete reason why we cannot customize our file tree to make it more obvious for the users (who are not necessarily Operator SDK experts, nor they should be required to be IMO) and to make our extended tools simpler and deterministic for the different deployment modes we want to support. If it's all adjustable by adapting the Makefile, then I don't see why not.

Right now, `make generate`, `make manifests` and pretty much all other necessary targets to build, install, deploy, locally or on whatever other Kubernetes server, all work. If `make bundle` is broken and there's no way to fix it with the manifests output to a dir other than `/config`, then this would be a concrete reason. Before assuming that though, I'll try to fix the `bundle` make target. If I cannot fix it, then I'll rollback to having all manifests under `/config`; otherwise, I still prefer separating install and deploy parts, until we are forced to rethink this, when we start using Operator bundles.

Reply:
:+1: